Dockerfile

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
+
+LABEL Maintainer="Olliver Schinagl <oliver@schinagl.nl>"
+
+WORKDIR /src
+
+COPY . /src/
+
+# hadolint ignore=DL3008  We want the latest stable versions
+RUN apk add --no-cache \
+        bsd-compat-headers \
+        bzip2-dev \
+        cmake \
+        curl-dev \
+        file \
+        fts-dev \
+        g++ \
+        git \
+        json-c-dev \
+        libmilter-dev \
+        libtool \
+        libxml2-dev \
+        linux-headers \
+        make \
+        ncurses-dev \
+        openssl-dev \
+        pcre2-dev \
+        python3 \
+        zlib-dev \
+    && \
+    mkdir -p "./build" && \
+    cd "./build" && \
+    cmake .. \
+          -DAPP_CONFIG_DIRECTORY="/etc/clamav" \
+          -DCMAKE_BUILD_TYPE"=release" \
+          -DCMAKE_INSTALL_FULL_BINDIR="/usr/bin" \
+          -DCMAKE_INSTALL_FULL_INCLUDEDIR="/usr/include" \
+          -DCMAKE_INSTALL_FULL_LIBDIR="/usr/lib" \
+          -DCMAKE_INSTALL_FULL_SBINDIR="/usr/sbin" \
+          -DCMAKE_INSTALL_LIBDIR="/usr/lib" \
```

Comment:
Please delete these lines. Setting `-DCMAKE_INSTALL_PREFIX="/usr"` should suffice.   I would also prefer if the clamav configs go to `/usr/etc/` instead of `/etc/clamav/` so it is closer to a standard clamav install.  Answering questions about customized install paths (or customized config names, in some cases) for distro packages is a small headache already. 

Except the build type one should be kept -- but fix the typo:
```
-DCMAKE_BUILD_TYPE="Release"
```

Reply:
I was not aware, I'm only familiar with distro pacakged versions :) but using the 'clamav' preferred way here, makes perfect sense and will do so.

and ... ouch ... :)

Reply:
Your point about the distro packages is well taken. I'm honestly less familiar with how each is customized so I went and checked.
- alpine: installs with standard names under `/etc/clamav/`
- debian & ubuntu: installs with standard names under `/etc/clamav/` and adds `/etc/clamav/onerrorexecute.d/` and `/etc/clamav/onupdateexecute.d/` for some reason. 🤷‍♂️
- centos & fedora: installs a `scan.conf` (instead of `clamd.conf`) under `/etc/clam.d/`  ☹
- freebsd: installs the same as default from source (`/usr/local/etc/`)

Your point re: single volume-mount is a great excuse to follow the example set by the alpine distro package.  Let's go with `/etc/clamav` then. Sorry for waffling on this. 

Let me know if you figure out the `lib64` issue.